### DISCUSSION TITLE
continuing training

### DISCUSSION THREAD

**Discussion by taborzbislaw** [MIC-DKFZ/nnUNet#900] [General]
Hi,

I am trying to continue training for BrainTumour dataset of Medical Segmentation Decathlon. 
After training for about 5000 epochs I reach the time limit of a queue system where I am doing computations so I have to restart training. I insert -c option to the command line. The training starts again but I got a strange behaviour as in the enclosed figure - it appears that the training is starting from scratch instead of starting from the latest checkpoint. What can be the cause of the problem?

Best regards,
Zbisław

![progress](https://user-images.githubusercontent.com/60366744/148207176-d11eaf1f-2f37-4af2-9bcb-7e4b5b4eba3b.png)

**Comment by FabianIsensee** [comment]
See #901 . Is there anything else we should discuss here? :-)

**Comment by taborzbislaw** [comment]
Hi,

no, everything is clear now. Thank you :)

Best regards,
Zbisław
